modules/stochastic_tools/include/utils/BootstrapCalculators.h

```diff
@@ -40,20 +40,36 @@ std::unique_ptr<const BootstrapCalculator> makeBootstrapCalculator(const MooseEn
 /*
  * Base class for computing bootstrap confidence level intervals. These classes follow the same
  * design pattern as those Statistics.h.
+ * @param other ParallelObject that is providing the Communicator
+ * @param levels The bootstrap confidence level intervals to compute in range (0, 1)
+ * @param replicates Number of bootstrap replicates to perform
+ * @param seed Seed for random number generator
  */
 class BootstrapCalculator : public libMesh::ParallelObject
 {
 public:
-  BootstrapCalculator(const libMesh::ParallelObject &,
-                      const std::vector<Real> &,
-                      unsigned int,
-                      unsigned int);
+  BootstrapCalculator(const libMesh::ParallelObject & other,
+                      const std::vector<Real> & levels,
+                      unsigned int replicates,
+                      unsigned int seed);
   virtual ~BootstrapCalculator() = default;
 
-  virtual std::vector<Real>
-  compute(const std::vector<Real> &, const Calculator &, const bool) const = 0;
+  /*
+   * Compute the bootstrap confidence level intervals.
+   * @param data Vector of data from which statistics are to be computed
+   * @param calc Calculator object defining the statistic to be computed
+   * @param is_distributed Flag indicating if the data is distributed in parallel
+   */
+  virtual std::vector<Real> compute(const std::vector<Real> & data,
+                                    const Calculator & calc,
+                                    const bool is_distributed) const = 0;
 
 protected:
+  // Compute Bootstrap estimates of a statistic
```

Comment:
Are we still requiring all methods and members to be doxygen commented?  If so, make these doxygen comments.

Reply:
the bootstrap stuff should be gone now, these were from other work that this was on top of.